src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -1036,6 +1039,116 @@ public async Task ReplaceClusterCert()
             RedirectToLeader();
         }
 
+        [RavenAction("/admin/certificates/localstate", "GET", AuthorizationStatus.Operator)]
+        public Task GetLocalState()
+        {
+            var includeSecondary = GetBoolValueQueryString("secondary", required: false) ?? false;
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var certificateList = new Dictionary<string, BlittableJsonReaderObject>();
+
+                try
+                {
+                    var localCertKeys = ServerStore.Cluster.GetCertificateThumbprintsFromLocalState(context).ToList();
+
+                    foreach (var localCertKey in localCertKeys)
+                    {
+                        var localCertificate = ServerStore.Cluster.GetLocalStateByThumbprint(context, localCertKey);
+                        if (localCertificate == null)
+                            continue;
+
+                        var def = JsonDeserializationServer.CertificateDefinition(localCertificate);
+
+                        if (includeSecondary || string.IsNullOrEmpty(def.CollectionPrimaryKey))
+                            certificateList.TryAdd(localCertKey, localCertificate);
+                        else
+                            localCertificate.Dispose();
+                    }
+
+                    using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                    {
+                        writer.WriteStartObject();
+                        writer.WriteArray(context, "Results", certificateList.ToArray(), (w, c, cert) =>
+                        {
+                            c.Write(w, cert.Value);
+                        });
+                        writer.WriteEndObject();
+                    }
+                }
+                finally
+                {
+                    foreach (var cert in certificateList)
+                        cert.Value?.Dispose();
+                }
+            }
+
+            return Task.CompletedTask;
+        }
+
+        [RavenAction("/admin/certificates/localstate/delete", "POST", AuthorizationStatus.ClusterAdmin)]
```

Comment:
local-state and change the action to `DELETE` ?

Reply:
yes